libmscore/scorefile.cpp

```diff
@@ -652,6 +670,43 @@ bool Score::saveFile(QFileInfo& info)
       return true;
       }
 
+//---------------------------------------------------------
+//   saveAvsOmrFile
+//    return true on success
+//---------------------------------------------------------
+#ifdef AVSOMR
+bool Score::saveAvsOmrFile(QIODevice *file, const QFileInfo& info, std::shared_ptr<Avs::AvsOmr> omr)
```

Comment:
I agree, moved to a separate place

Reply:
Well, actually I proposed this idea to avoid a dependency of `libmscore` on `avsomr` module, but the dependency is still remaining. Would it be possible to avoid this dependency as well? Apart from saving `.msmr` files there don't seem to be reasons to introduce this dependency.

Reply:
At the moment, we can not avoid, but it will be possible in the future. If you replace the use of the AvsOmr object with the use of the interface, and receive the implementation of this interface through the IoC (what Vasily Pereverzev began to do). At this stage, this requires significant changes, so it will be in the future.